src/views/Pipeline.vue

```diff
@@ -502,7 +503,11 @@ export default defineComponent({
         event: ev,
         componentProps: { job }
       });
-      return popover.present()
+      emitter.on("unSelectPinJob", (enumId)=> {
```

Comment:
We should pass the function reference as it will be needed for unsubscribe

Reply:
Improved sir